packages/framework/src/types/step.types.ts

```diff
@@ -173,47 +173,62 @@ export type ChannelStep<
 ) => StepOutput<T_Result>;
 
 export type EmailOutput = FromSchema<(typeof channelStepSchemas)[ChannelStepEnum.EMAIL]['output']>;
+export type EmailOutputUnvalidated = FromSchemaUnvalidated<
+  (typeof channelStepSchemas)[ChannelStepEnum.EMAIL]['output']
+>;
```

Comment:
I am adding these `Unvalidated` types as new types because most of the time, type consumers of `@novu/framework` will need the validated types which don't have the `Unvalidated` suffix. We already have a number of places using the validated types, so I've opted not to update them both for the stated reason above regarding usage, and to avoid a large number of file changes.